pkg/imageserver/webdav.go

```diff
@@ -101,6 +101,15 @@ func (s *webdavImageServer) GetHandler(meta *iiapi.InspectorMetadata,
 		w.Write(body)
 	})
 
+	mux.HandleFunc(s.opts.ResultAPIUrlPath, func(w http.ResponseWriter, r *http.Request) {
+		resultJSON, err := json.Marshal(results)
```

Comment:
Why not use `json.MarshalIndent` like the other endpoints?

Reply:
fixed